config.yaml

```diff
@@ -1,8 +1,4 @@
 options:
-  metrics-port:
-    type: string
-    default: '8080'
-    description: Metrics port
```

Comment:
We are sure that people shouldn't be able to configure in which port metrics are served, right?

Reply:
I decided to remove it because even if people could change this port in the charm's configuration, internally we won't be exposing the correct port in the k8s `Service` linked to this `Pod` because we do not have any mechanism in place (atm) for doing it dynamically.
We could discuss this with the team as well, but in general I don't think this will be changed quite often.

Reply:
I don't expect this to need to be configured quite often too. Could you add a note to discuss with the team and move on from there? Maybe we could also standardize this during the observability bundle effort.

Reply:
+1